### PR TITLE
More hit data

### PR DESCRIPTION
**Pull request by yramachers**
Slight misnomer, my fault. That was the initial intention but it turns out that we already get the maximum information about the Ge77 production, with the energy scorer being rather redundant. Geant4 does not distinguish between Ge77 and its isomer, the neutron capture cross sections is a total, i.e. for both. That explains Fig. 4 in the Gerda paper where they have to discuss the split between ground-state and isomer production with measured cross sections.
In summary, this PR contains the removal of radioactive decay physics which isn't needed in this application and a more important relaxing of the neutron time cut to 2 milliseconds which is the upper limit from the Gerda paper. The latter leads to enhanced Ge77 production numbers due to allowing slower, more thermal neutrons.
This PR should probably bump up the version number since production runs could start from this, well sort of have already for testing.

### CODE REVIEW

File: src/WLGDPSTrackWeight.cc [hunk 1]
```diff
@@ -21,8 +21,8 @@ WLGDPSTrackWeight::~WLGDPSTrackWeight() = default;
 
 G4bool WLGDPSTrackWeight::ProcessHits(G4Step* aStep, G4TouchableHistory* /*unused*/)
 {
-  if(aStep->GetTotalEnergyDeposit() == 0.)
-    return false;  // not for zero edep
+//  if(aStep->GetTotalEnergyDeposit() == 0.)
+//    return false;  // not for zero edep
```
Comment (drbenmorgan) [inline src/WLGDPSTrackWeight.cc:25]:
```suggestion
```

File: warwick-legend.cc [hunk 2]
```diff
@@ -78,9 +79,15 @@ int main(int argc, char** argv)
 
   // -- set user physics list
   auto* physicsList = new Shielding;
+
+  // remove radioactive decay from list
+  // auto* removeFromList = new G4RadioactiveDecayPhysics();
+  // physicsList->RemovePhysics(removeFromList);
+
```
Comment (drbenmorgan) [inline warwick-legend.cc:86]:
```suggestion
```

**Review by drbenmorgan — CHANGES_REQUESTED** [review]

**Review by drbenmorgan — APPROVED** [review]

**Comment by drbenmorgan** [comment]
Thanks @yramachers, I'll merge, but do you still want the version number to be incremented? We can also consider tags at the point.
